source/plugin/commands/arguments.rst

```diff
@@ -25,13 +25,15 @@ it with :javadoc:`CommandContext#getOne(String)`. Optional and weak arguments ma
 
 Many of the parsers may return more than one object (e.g. multiple players with a matching username). In that case, you
 must use the :javadoc:`CommandContext#getAll(String)` method to get the ``Collection`` of possible matches.
-**Otherwise, the context object will throw an exception!**
+**Otherwise, the context object will throw an exception!** Please make sure that you make a conscious decision whether
+you need to limit your ``CommandExecutor`` to a single result or can support multiple arguments. If you support multiple
+arguments the users have to type less and shorter commands. Example: ``/tell @a Who took the cookies?``.
```

Comment:
"have to type less and shorter commands" could be "can use a shorter command syntax."